server/pbench/bin/index-pbench

```diff
@@ -676,24 +702,30 @@ class ToolData(object):
 
         self.csv_files = None
         self.json_files = None
+        self.stdout_files = None
```

Comment:
Would you be willing to perform some additional refactoring to smooth this out a bit?

Can we merge all the json, csv, and output, handlers into one structure, `_known_handlers`, indexed by tool?  Then have one `staticmethod` called, `get_files` which picks the correct `get_*_files` method based on the handler in play?

Reply:
This kind of simplification forces me to change code that this PR did not already alter, such as the unified csv handling and the json handling. I don't want to break anything, but will comply with the request.